codalab/apps/teams/migrations/0005_auto_20220822_1451.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-08-22 14:51
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import storages.backends.s3boto3
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('teams', '0004_auto_20220524_1519'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='team',
+            name='image',
+            field=models.FileField(blank=True, null=True, storage=storages.backends.s3boto3.S3Boto3Storage(bucket='lri-public'), upload_to='team_logo', verbose_name='Logo'),
```

Comment:
Where did this migration come from? What is being changed?

Is your `.env` pointing to the new storage class?
`DEFAULT_FILE_STORAGE=apps.web.utils.CodalabS3BotoStorage`

Reply:
Indeed it did not. Nice catch !
I'm updating it.